### PR TITLE
Adding WCT as a Status Enum

### PR DESCRIPTION
**Pull request by joelridden**
As part of the workflow metadata refactor, the WCT needed to be added to the Status Enum

### CODE REVIEW

**Review by jasonmotha — DISMISSED** [review]
Pending https://github.com/ucgmsim/slurm_gm_workflow/pull/433#event-6322770019 being merged at the same time

**Review by sungeunbae — COMMENTED** [review]
WCT just refers to "wall clock time", not directly indicating the status. what about timesup or an adjective carrying similar meaning, rather than a noun?

**Review by jpa198 — DISMISSED** [review]

File: qcore/constants.py [hunk 1]
```diff
@@ -274,7 +274,8 @@ class Status(ExtendedStrEnum):
     running = 3, "running"
     unknown = 4, "unknown"
     completed = 5, "completed"
-    failed = 6, "failed"
+    killed_WCT = 6, "killed_WCT"
```
Comment (jpa198) [inline qcore/constants.py:277]: Excellent, a good choice of prefix

**Review by jasonmotha — APPROVED** [review]
Good call to add the suffix after we decided on slaughtered / killed.

**Review by sungeunbae — APPROVED** [review]

**Pull request merged**